nftables.go

```diff
@@ -29,6 +33,9 @@ func newNFTables(config *bouncerConfig) (interface{}, error) {
 	if !config.DisableIPV6 {
 		ret.conn6 = &nftables.Conn{}
 	}
+	ret.DenyAction = config.DenyAction
```

Comment:
Do we want to use `strings.Upper(config.DenyAction)` to improve user experience? (so upper and lower case word will be accepted)